satpy/readers/vii_base_nc.py

```diff
@@ -0,0 +1,236 @@
+
+        try:
+            longitude = self[filetype_info['cached_longitude']]
+            latitude = self[filetype_info['cached_latitude']]
+
+            if self.interpolate:
+                self.longitude, self.latitude = self._perform_geo_interpolation(longitude, latitude)
+            else:
+                self.longitude, self.latitude = longitude, latitude
+
+        except KeyError:
+            logger.warning("Cached longitude and/or latitude datasets are not correctly defined in YAML file")
+            self.longitude, self.latitude = None, None
+
+    def get_dataset(self, dataset_id, dataset_info):
+        """Get dataset using file_key in dataset_info."""
+        var_key = dataset_info['file_key']
+        logger.debug('Reading in file to get dataset with key %s.', var_key)
+
+        if var_key == 'cached_longitude' and self.longitude is not None:
+            variable = self.longitude.copy()
+        elif var_key == 'cached_latitude' and self.latitude is not None:
+            variable = self.latitude.copy()
+        else:
+            try:
+                variable = self[var_key]
+            except KeyError:
+                logger.warning("Could not find key %s in NetCDF file, no valid Dataset created", var_key)
+                return None
+
+            # If the dataset is marked for interpolation, perform the interpolation from tie points to pixels
+            if dataset_info.get('interpolate', False) and self.interpolate:
+                variable = self._perform_interpolation(variable)
+
+            # Perform the calibration if required
+            if dataset_info['calibration'] is not None:
+                variable = self._perform_calibration(variable, dataset_info)
+
+        # Perform the orthorectification if required
+        if self.orthorect:
+            orthorect_data_name = dataset_info.get('orthorect_data', None)
+            if orthorect_data_name is not None:
+                variable = self._perform_orthorectification(variable, orthorect_data_name)
+
+        # If the dataset contains a longitude, change it to the interval [0., 360.) as natively in the product
+        # since the unwrapping performed during the interpolation might have created values outside this range
+        if dataset_info.get('standard_name', None) == 'longitude':
+            variable = self._perform_wrapping(variable)
+
+        # Manage the attributes of the dataset
+        variable.attrs.setdefault('units', None)
+
+        variable.attrs.update(dataset_info)
+        variable.attrs.update(self._get_global_attributes())
+
+        return variable
+
+    @staticmethod
+    def _perform_wrapping(variable):
+        """Wrap the values to the interval [0, 360.) .
+
+        Args:
+            variable: xarray DataArray containing the dataset to wrap.
+
+        Returns:
+            DataArray: array containing the wrapped values and all the original metadata
+
+        """
+        variable %= 360.
+        return variable
```

Comment:
This function is used once and is just a wrapper around `%= 360.` Why is this needed ?